### PR TITLE
docs: add 1.7 changelog and update instructions

### PR DESCRIPTION
**Pull request by alexmt**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,50 @@
 # Changelog
 
+## v1.7.0 (Unreleased)
+
+### GnuPG Signature Verification
+
+The feature allows to only sync against commits that are signed in Git specified GnuPG. The list of public
```
Comment (jannfis) [inline CHANGELOG.md:7]: `specified` -> `using`

Comment (jannfis) [inline CHANGELOG.md:7]: `specified` -> `using`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,50 @@
...
+The feature allows to only sync against commits that are signed in Git specified GnuPG. The list of public 
+GPG keys required for verification is configured at the system level and can be managed using Argo CD CLI or Web user interface.
+The keys management is integrated with Argo CD SSO and access control system (e.g. `argocd gpg add --from <path-to-key>`)
+
+The signature verification is enabled on the project level. The ApplicationProject CRD has a new signatureKeys field that includes
+a list of imported public GPG keys. Argo CD will verify the commit signature by these keys for every project application.
+
+### Cluster Management Enhancements
+
+Feature allows using the cluster name instead of the URL to specify the application destination cluster.
```
Comment (jannfis) [inline CHANGELOG.md:16]: `Feature` -> `The feature`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,50 @@
...
+as the number of monitored Kubernetes resources.
+
+### Diffing And Synchronization Usability
+
+* **Diffing logic improvement** Argo CD performs client-side resource diffing to detect deviations and present detected
+differences in the UI and CLI. The 1.7 release aligns a comparison algorithm with server-side Kubernetes implementation
+and removes inaccuracies in some edge cases.
+
+* **Helm Hooks Compatibility** The improvement removes the discrepancy between the way how Argo CD and Helm deletes
+hooks resources. This significantly improves the compatibility allows and enables additional use cases.
```
Comment (jannfis) [inline CHANGELOG.md:28]: `allows and enables additional use cases` -> `enables additional use cases`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,50 @@
...
+
+* **Namespace Auto-Creation** You can click a checkbox and Argo CD will ensure that namespace specified as the
```
Comment (jannfis) [inline CHANGELOG.md:30]: `You can click a checkbox and` -> `With a new option for applications, ` or something along these lines maybe?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,50 @@
...
+application destination exists in the destination cluster. 
+
+* **Failed Sync Retry** This feature enables retying of failed synchronization attempts during both manually-triggered
```
Comment (jannfis) [inline CHANGELOG.md:33]: typo: `retying` -> `retrying`

**Review by jannfis — CHANGES_REQUESTED** [review]
Found some typos and some phrasing to be discussed.

File: CHANGELOG.md (hunk 1 above)
Comment (alexmt) [reply CHANGELOG.md:7]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (alexmt) [reply CHANGELOG.md:16]: fixed

File: CHANGELOG.md (hunk 3 above)
Comment (alexmt) [reply CHANGELOG.md:28]: fixed

File: CHANGELOG.md (hunk 4 above)
Comment (alexmt) [reply CHANGELOG.md:30]: I like your proposal! Used it

**Comment by alexmt** [comment]
Thank you for review @jannfis ! PTAL

File: CHANGELOG.md (hunk 5 above)
Comment (jannfis) [reply CHANGELOG.md:33]: You forgot this typo :)

**Review by jannfis — APPROVED** [review]
LGTM except that typo

File: CHANGELOG.md (hunk 5 above)
Comment (alexmt) [reply CHANGELOG.md]: thank you!

**Pull request merged**